<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InGameDialog</class>
 <widget class="QDialog" name="InGameDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GAME NAME HERE â€“ NOT THE SAME NAME</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartGame">
       <property name="text">
        <string>Start Game</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Players in game:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="wTwoHeadedGiant" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Team 1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="lvTeam1"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="btnSwitchTeams">
        <property name="text">
         <string>Switch Teams</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Team 2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="lvTeam2"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QListView" name="lvPlayers"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>InGameDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>92</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>513</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
